tests/datavolume_test.go

```diff
@@ -1119,6 +1119,24 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			table.Entry("[test_id:8045]for clone DataVolume", createCloneDataVolume, fillCommand),
 		)
 
+		It("Should handle a pre populated DV", func() {
+			By(fmt.Sprintf("initializing dataVolume marked as prePopulated %s", dataVolumeName))
+			dataVolume := utils.NewDataVolumeWithHTTPImport(dataVolumeName, "1Gi", cirrosURL())
+			dataVolume.Annotations["cdi.kubevirt.io/storage.prePopulated"] = dataVolumeName
+
+			dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
+			Expect(err).ToNot(HaveOccurred())
+
+			Eventually(func() cdiv1.DataVolumePhase {
+				dv, _ := f.CdiClient.CdiV1beta1().DataVolumes(dataVolume.Namespace).Get(context.TODO(), dataVolume.Name, metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				return dv.Status.Phase
+			}, timeout, pollingInterval).Should(Equal(cdiv1.Succeeded))
```

Comment:
If the PVC does not exist, this should not be Succeeded

Reply:
Thanks for review. 

Yes, I have started this PR before the discussions on the behavior, and realizing that it is better to use real statuses like `Pending ` and `Succeeded`. That is why I made it Work In Progress.

Now I can go back to work and make it production ready.

Reply:
Done, now it is in Pending, and transitions to Succeeded when PVC is available